src/views/Pricing/Pricing.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { Heading, Toggler } from '@konturio/ui-kit';
+import clsx from 'clsx';
+import { useAtom } from '@reatom/react-v2';
+import { CURRENT_SUBSCRIPTION_INFO, MOCK_PRICING_CONFIG } from '~views/Pricing/mock';
+import PaymentPlan from '~features/subscriptions/components/PaymentPlanCard/PaymentPlan';
+import { i18n } from '~core/localization';
+import { BillingCycleID } from '~features/subscriptions/types';
+import { FeatureFlag, featureFlagsAtom } from '~core/shared_state';
+import { getCurrentUserSubscription } from '~core/api/subscription';
+import s from './Pricing.module.css';
+import type { PricingConfig } from '~views/Pricing/types';
+import type { CurrentSubscriptionInfo } from '~features/subscriptions/types';
+
+const togglerInitialValue = BillingCycleID.Annually;
+
+export function PricingPage() {
+  const [featureFlags] = useAtom(featureFlagsAtom);
+  const config: PricingConfig =
+    featureFlags[FeatureFlag.SUBSCRIPTION]?.configuration || MOCK_PRICING_CONFIG;
+
+  const [subscriptionData, setSubscriptionData] = useState<CurrentSubscriptionInfo>(null);
+
+  const [currentBillingCycleID, setCurrentBillingCycleID] =
+    useState<BillingCycleID>(togglerInitialValue);
+
+  const [offTogglerConfig, onTogglerConfig] = config.billingCyclesDetails;
```

Comment:
`offTogglerConfig` and `onTogglerConfig` is not saying anything about the actual data they store. Maybe `monthlyPlansConfig` and `yearlyPlansConfig` would be better